internal/pkg/config/config.go

```diff
@@ -29,10 +29,11 @@ var (
 
 // InjectionConfig is a specific instance of a injected config, for a given annotation
 type InjectionConfig struct {
-	Name        string             `json:"name"`
-	Containers  []corev1.Container `json:"containers"`
-	Volumes     []corev1.Volume    `json:"volumes"`
-	Environment []corev1.EnvVar    `json:"env"`
+	Name         string               `json:"name"`
+	Containers   []corev1.Container   `json:"containers"`
+	Volumes      []corev1.Volume      `json:"volumes"`
+	Environment  []corev1.EnvVar      `json:"env"`
+	VolumeMounts []corev1.VolumeMount `json:"volumeMounts"`
```

Comment:
preference for no camel-case in json keys.`volume-mounts` instead?

Reply:
Fine for me, I just used it, because I'm used to camelCase in Kubernetes Manifests :+1:

Reply:
yea, understandable

Reply:
soo... do we leave it like it is to go with the "usual Kubernetes YAML feeling" (=camelCase) or convert to lower-case with dashes `-`?

Reply:
id rather use dashes; it feels more idiomatic in native JSON/YAML configs. And, this isnt a kubernetes resource; merely kubernetes-adjacent :P

Reply:
Alright, changed it :)